### PR TITLE
Check all properties with property option

### PR DESCRIPTION
**Pull request by peterschrammel**
addresses https://github.com/diffblue/cbmc/issues/254

### CODE REVIEW

File: src/util/cmdline.cpp [hunk 1]
```diff
@@ -220,6 +220,38 @@ const std::list<std::string>& cmdlinet::get_values(const std::string &option) co
...
+      values.push_back(value.substr(idx, next-idx));
+      if(next==std::string::npos) break;
+      idx=next;
+    }
+  }
+  return values;
+}
+
+/*******************************************************************\
+
```
Comment (tautschnig) [inline src/util/cmdline.cpp]: We meanwhile have `split_string` in util/string_utils.h merged, hence that function shouldn't be necessary anymore.

File: src/cbmc/cbmc_parse_options.cpp [hunk 2]
```diff
@@ -595,10 +593,12 @@ bool cbmc_parse_optionst::set_properties(goto_functionst &goto_functions)
...
   {
     if(cmdline.isset("claim")) // will go away
-      ::set_properties(goto_functions, cmdline.get_values("claim"));
+      ::set_properties(goto_functions, 
+                       cmdline.get_comma_separated_values("claim"));
 
     if(cmdline.isset("property")) // use this one
-      ::set_properties(goto_functions, cmdline.get_values("property"));
+      ::set_properties(goto_functions, 
+                       cmdline.get_comma_separated_values("property"));
```
Comment (tautschnig) [inline src/cbmc/cbmc_parse_options.cpp]: Please update code formatting.

**Comment by peterschrammel** [comment]
@kroening, do we still need `--property prop1,prop2,...propn`? Or are they happy with `--property prop1 --property prop2` etc? In case of the latter, we can close this PR.

**Comment by peterschrammel** [comment]
Parts of this PR have been cherry-picked. The rest seems obsolete.

**Pull request closed without merging**